tsl/src/continuous_aggs/insert.c

```diff
@@ -155,6 +155,14 @@ cache_inval_entry_init(ContinuousAggsCacheInvalEntry *cache_entry, int32 hyperta
 	/* NOTE: we can remove the id=>relid scan, if it becomes an issue, by getting the
 	 * hypertable_relid directly from the Chunk*/
 	Hypertable *ht = ts_hypertable_cache_get_entry_by_id(ht_cache, hypertable_id);
+	if (ht == NULL)
+	{
+		ereport(ERROR,
+				(errcode(ERRCODE_UNDEFINED_OBJECT),
+				 errmsg("unable to get relid for hypertable %d in cache_inval_entry_init",
+						hypertable_id)));
```

Comment:
```suggestion
				(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
				 errmsg("invalid hypertable",
						)));
```

Reply:
I changed the error code as suggested and shortened the error message.